zebra-chain/src/parameters/arbitrary.rs

```diff
@@ -18,4 +18,17 @@ impl NetworkUpgrade {
         ]
         .boxed()
     }
+
+    /// Generates network upgrades from a reduced set
+    pub fn reduced_branch_id_strategy() -> BoxedStrategy<NetworkUpgrade> {
```

Comment:
This can probably be done directly in https://github.com/ZcashFoundation/zebra/blob/ee9294a821e202724f3fdcc759cbab2661387d52/zebra-state/src/service/arbitrary.rs#L108
Unsure if that will be a better choice.

Reply:
I think it's fine, we're going to revise this code in #1974 anyway.